cypress/integration/PressReleases/index.js

```diff
@@ -0,0 +1,63 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+And('the 2019 release doesnt have a link', () => {
+    cy.get('p').should('contain.text', '2019').should('not.have','a');
+});
+
+And('the following press releases are displayed', (dataTable) => {
+    for (const { title, link } of dataTable.hashes()) {
+        cy.get('p').find(`a:contains('${title}')`).should('have.attr', 'href', link);
+    }
+});
+
+And('{string} box is displayed with a link {string}', (linkText, linkHref) => {
+    cy.get('.callout-box-full').find('h3').should('have.text', linkText);
+    cy.get('.callout-box-full').find('a').should('have.attr', 'href', linkHref);
+    
+});
+
+And('the list of press release pages is displayed', () => {
+    cy.get('.views-element-container').should('exist');
```

Comment:
replace exist with be.visible ( if it is not displayed for whatever reason, it wont catch it)

Reply:
updated this.